draft-ietf-dnsop-svcb-https.md

```diff
@@ -1228,12 +1228,18 @@ Section 12.1 of HSTS}}.
 
 ## Use of HTTPS RRs in other protocols
 
-All protocols employing "http://" or "https://" URLs SHOULD respect HTTPS RRs.
-For example, clients that
+All HTTP connections to named origins are eligible to use HTTPS RRs, even
+when HTTP is used as part of another protocol or without an explicit HTTP
+URL.  For example, clients that
 support HTTPS RRs and implement the altered WebSocket {{!WebSocket=RFC6455}}
 opening handshake from the W3C Fetch specification {{FETCH}} SHOULD use HTTPS RRs
 for the `requestURL`.
 
+When HTTP is used in a context where URLs or redirects are not applicable
+(e.g. Proxy Auto-Configuration), clients MUST implement transport security
```

Comment:
This is confusing because the PAC fetch itself is just regular URL-based http/https fetches. Redirects work naturally there.

What the bug refers to isn't the PAC fetch, but the proxy config itself. The proxy config may have been the output of a PAC script (which I mention in the bug because it's one of the few places where proxy descriptors are part of some interoperable thing, rather than some implementation-specific scheme). It's of interest because it doesn't specify a URL. But only sort of because, if you did specify a URL, redirects are still not applicable.

Perhaps just "e.g. connections to an HTTP(S) proxy".

Reply:
OK, reworded.